From bee22b310ce2ac3dc3e0a334b423bbc810417a60 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Sun, 5 Jan 2020 14:05:55 +0100
Subject: [PATCH] [devscripts/create-github-release] do not import base64

Without this change `make offlinetest` fails with:

	./devscripts/create-github-release.py:4:1: F401 'base64' imported but unused
---
 devscripts/create-github-release.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/devscripts/create-github-release.py b/devscripts/create-github-release.py
index 6464ef322..2ddfa1096 100644
--- a/devscripts/create-github-release.py
+++ b/devscripts/create-github-release.py
@@ -1,7 +1,6 @@
 #!/usr/bin/env python
 from __future__ import unicode_literals
 
-import base64
 import io
 import json
 import mimetypes
